examples/simple-non-wasm/README.md

```diff
@@ -0,0 +1,81 @@
+# simple-non-wasm
+
+## Setup
+
+
+```bash
+➜ tree . -I 'target/'                  
+.
+├── Cargo.lock
+├── Cargo.toml
+├── README.md
+├── config
+│   ├── config.yaml
+│   └── manifest.yaml
+├── programs
+│   ├── counter
+│   │   ├── Forc.lock
+│   │   ├── Forc.toml
+│   │   ├── out
+│   │   │   └── debug
+│   │   │       ├── counter-abi.json
+│   │   │       └── counter.bin
+│   │   └── src
+│   │       └── main.sw
+│   └── counter-rs
+├── schema
+│   └── counter.graphql
+└── src
+    ├── lib.rs
+    └── main.rs
+
+9 directories, 13 files
+```
+
+## Usage
+
+#### Run migrations
+  - In this example we're using an `indexer` database owned by the `postgres` user without a password
+
+```bash
+DATABASE_URL="postgres://postgres@127.0.0.1:5432/indexer" diesel migration list --migration-dir=./schema/migrations
```

Comment:
See https://github.com/FuelLabs/fuel-indexer/issues/54 if you run into issues here